nemo/collections/asr/models/clustering_diarizer.py

```diff
@@ -90,7 +90,8 @@ def __init__(self, cfg: DictConfig):
 
         # init speaker model
         self._init_speaker_model()
-        self._num_speakers = self._cfg.diarizer.num_speakers
+        self._num_speakers = self._cfg.diarizer.oracle_num_speakers
```

Comment:
Keep backward compatibility with older checkpoints here.

Reply:
Sure :)

Reply:
Actually we don't publish diarizer checkpoints but yes I will handle that case anyhow